tests/loan-token/BorrowingTestToken.test.js

```diff
@@ -49,22 +56,26 @@ const hunEth = new BN(wei("100", "ether"));
 
 contract("LoanTokenBorrowing", (accounts) => {
     let owner, account1;
-    let sovryn, SUSD, WRBTC, RBTC, BZRX, loanToken, loanTokenWRBTC, SOV, priceFeeds;
+    let sovryn, SUSD, TestERC777, WRBTC, RBTC, BZRX, loanToken, loanTokenWRBTC, SOV, priceFeeds;
 
     async function deploymentAndInitFixture(_wallets, _provider) {
         SUSD = await getSUSD();
+        TestERC777 = await getERC777(owner);
         RBTC = await getRBTC();
         WRBTC = await getWRBTC();
         BZRX = await getBZRX();
         priceFeeds = await getPriceFeeds(WRBTC, SUSD, RBTC, BZRX);
 
         sovryn = await getSovryn(WRBTC, SUSD, RBTC, priceFeeds);
 
-        loanToken = await getLoanToken(owner, sovryn, WRBTC, SUSD);
-        loanTokenWRBTC = await getLoanTokenWRBTC(owner, sovryn, WRBTC, SUSD);
+        loanToken = await getLoanToken(owner, sovryn, WRBTC, SUSD, true);
+        loanTokenWRBTC = await getLoanTokenWRBTC(owner, sovryn, WRBTC, SUSD, true);
         await loan_pool_setup(sovryn, owner, RBTC, WRBTC, SUSD, loanToken, loanTokenWRBTC);
 
         SOV = await getSOV(sovryn, priceFeeds, SUSD, accounts);
+
+        // need to deploy the mutex in the initialization. Otherwise, the global reentrancy prevetion will not be working & throw an error
```

Comment:
```suggestion  
        // Need to deploy the mutex in the initialization. Otherwise, the global reentrancy prevention will not be working & throw an error.
```